### PR TITLE
Opprett tabell for tiltakshendelser

### PR DESCRIPTION
**Pull request by dragutin-nav**
## Describe your changes

Create table for tiltakshendelser

## Trello ticket number and link

https://trello.com/c/829ZX8io/667-create-service-and-repository-that-will-handle-vaas-data

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [✔️] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [✔️] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] We need to implement grafana analytics

### CODE REVIEW

File: src/main/resources/db/postgres/V1_89__vaas.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE VAAS
+(
+    hendelse_id      varchar(20)  NOT NULL PRIMARY KEY,
+    ident            VARCHAR(20)  NOT NULL,
```
Comment (JulieHillRoa) [inline src/main/resources/db/postgres/V1_89__vaas.sql:4]: kunne vi enten satt fnr eller 11 på varchar? synes det er så stress å måtte logge inn i db og måtte hente ut rader for å skjønne hva vi bruker på personen 🙈

**Review by klaramargrethehelgemo — CHANGES_REQUESTED** [review]
Skriv enten lenke eller link, det blir riktig med lenke her.

File: src/main/resources/db/postgres/V1_89__vaas.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE VAAS
```
Comment (JulieHillRoa) [inline src/main/resources/db/postgres/V1_89__vaas.sql:1]: kanskje et annet navn.. noe mer liknende topic-navnet?

Comment (dragutin-nav) [reply src/main/resources/db/postgres/V1_89__vaas.sql:1]: I don't think that it should be exactly like topic name. Maybe just `handelser`?

File: src/main/resources/db/postgres/V1_89__vaas.sql (hunk 1 above)
Comment (dragutin-nav) [inline src/main/resources/db/postgres/V1_89__vaas.sql:4]:
```suggestion
    fnr            VARCHAR(11)  NOT NULL,
```

**Review by ingfo — APPROVED** [review]

**Pull request merged**